packages/design-system/src/components/WIP/Accordion/Accordion.stories.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import { Story } from '@storybook/react';
+
+import { TagSuccess } from '../../Tag';
+
+import Accordion from './Accordion';
+import CollapsiblePanel from './Primitive/CollapsiblePanel';
+import { StackVertical } from '../../Stack';
+
+export default { component: Accordion };
+
+const SampleParagraph = () => (
+	<p>
+		Quisque efficitur, magna sit amet tempor malesuada, orci mauris vestibulum enim, quis gravida
+		est urna et ipsum. Nunc rutrum, magna id fermentum dignissim, magna sem volutpat risus, ut
+		ultrices ipsum lacus vitae sapien. Curabitur sodales risus ac nibh efficitur, dapibus posuere
+		ipsum bibendum. Proin erat ipsum, tempus in aliquet sed, auctor id sem. Maecenas ultrices, magna
+		vitae pretium condimentum, ipsum lectus hendrerit est, a ultrices lacus odio in mi. Phasellus
+		accumsan diam in metus dictum ultrices. In hac habitasse platea dictumst. Curabitur vestibulum
+		vitae libero sit amet blandit. Nulla bibendum sollicitudin dolor at vehicula. Morbi quis viverra
+		velit, eget ornare velit. Praesent porttitor sagittis nulla non vehicula. u
+	</p>
+);
+
+export const SimpleCollapsiblePanel = () => (
+	<div style={{ maxWidth: '80rem', marginLeft: 'auto', marginRight: 'auto', padding: '3rem' }}>
+		<CollapsiblePanel a11yId="simpleCollapsible" title="Simple panel">
+			<SampleParagraph />
+		</CollapsiblePanel>
+	</div>
+);
+
+export const DisabledPanel = {
+	render: (props: Story) => (
+		<div style={{ maxWidth: '80rem', marginLeft: 'auto', marginRight: 'auto', padding: '3rem' }}>
+			<CollapsiblePanel
+				{...props}
+				a11yId="disabledCollapsible"
+				title="disabled panel"
+				action={{
+					icon: 'plus',
+					tooltip: 'action tooltip',
+					callback: () => window.alert('action callback'),
+				}}
+				disabled
+			>
+				<SampleParagraph />
+			</CollapsiblePanel>
+		</div>
+	),
+};
+
+export const SmallPanel = () => (
+	<div style={{ maxWidth: '80rem', marginLeft: 'auto', marginRight: 'auto', padding: '3rem' }}>
+		<CollapsiblePanel a11yId="smallCollapsible" title="small panel" size="S">
+			<SampleParagraph />
+		</CollapsiblePanel>
+	</div>
+);
+
+export const WithMetadata = () => (
```

Comment:
This story also needs to showcase string metadata, and single component metadata (to show what happens when there's no need for separators)

Reply:
I added two more example.
WDYT ?